trulens_eval/examples/snowflake_demo/feedback.py

```diff
@@ -10,33 +11,46 @@
 
 load_dotenv()
 
-db_url = 'snowflake://{user}:{password}@{account}/{dbname}/{schema}?warehouse={warehouse}&role={role}'.format(
-    user=os.environ['SF_USER'],
-    account=os.environ['SF_ACCOUNT'],
-    password=os.environ['SF_PASSWORD'],
-    dbname=os.environ['SF_DB_NAME'],
-    schema=os.environ['SF_SCHEMA'],
-    warehouse=os.environ['SF_WAREHOUSE'],
-    role=os.environ['SF_ROLE'],
+db_url = "snowflake://{user}:{password}@{account}/{dbname}/{schema}?warehouse={warehouse}&role={role}".format(
+    user=os.environ["SF_USER"],
+    account=os.environ["SF_ACCOUNT"],
+    password=os.environ["SF_PASSWORD"],
+    dbname=os.environ["SF_DB_NAME"],
+    schema=os.environ["SF_SCHEMA"],
+    warehouse=os.environ["SF_WAREHOUSE"],
+    role=os.environ["SF_ROLE"],
 )
 
 tru = Tru(database_url=db_url)
 
 provider = LiteLLM(model_engine="replicate/snowflake/snowflake-arctic-instruct")
+small_local_model_provider = SmallLocalModels(
+    os.getenv(
+        "SMALL_LOCAL_MODELS_CONTEXT_RELEVANCE_MODEL_PATH",
+        "/trulens_demo/small_local_models/context_relevance",
```

Comment:
LGTM - could you remind me of how our private HF model is provisioned to the demo deployment

Reply:
oh I'm going to store it locally when I create the docker image.